e2e/tests/cookie-banner.spec.ts

```diff
@@ -58,27 +80,81 @@ test.describe('cookie banner', () => {
   })
 
   test.describe('cookie preferences has been set', () => {
```

Comment:
Could help to make to name of the describe a bit more specific:
```suggestion
  test.describe('cookie preferences has been previously set', () => {
```